environment.ci.yml

```diff
@@ -4,9 +4,13 @@ channels:
     - conda-forge
 dependencies:
     - codecov
-    - flake8
+    - flake8==4.0.1
```

Comment:
Why do we lock it to version `4.0.1`?

Reply:
Because in the pre-commit configuration file, we have to choose a version. To avoid to have different version of flake8, I lock to to the same version.